### DISCUSSION TITLE
Could we refine the menu settings to make it a primary menu option?

### DISCUSSION THREAD

**Discussion by rayangl** [dani-garcia/vaultwarden#5104] [Ideas]
![image](https://github.com/user-attachments/assets/b015819a-3968-4d9c-89d0-60c70024c8ba)
![image](https://github.com/user-attachments/assets/4437cfe3-c08f-4cd9-b7e8-0c9acd655d38)
Is it feasible to position the multi-select menu more prominently? If space is limited, the relevant buttons could be displayed upon selecting multiple items.

**Comment by BlackDex** [comment]
This project does not maintain or develop any client, including the web-vault. Only small changes are done to make Vaultwarden work.
